index.html

```diff
@@ -484,6 +484,11 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 </li>
 <li>zero, otherwise.</li>
 </ol>
+<p class="note">Note that according to the definition of <a
+data-cite="HR-TIME-2#dfn-time-origin">time origin</a> in workers, it is possible
+and expected that certain attributes of <a
+data-cite="service-workers-nightly#dom-serviceworkerregistration-navigationpreload">
+navigation preload</a> requests may include negative timestamps.</p>
```

Comment:
s/may include negative timestamps/will be negative `DOMHighResTimeStamp`s

Reply:
maybe "it is possible that some attributes of navigation preload requests will contain negative DOMHighResTimeStamp values"?

Reply:
or:
s/contain/have
"it is possible that some attributes of navigation preload requests will have negative DOMHighResTimeStamp values"

Reply:
sold!